app/assets/stylesheets/resources.css.scss

```diff
@@ -0,0 +1,89 @@
+.resources {
+  list-style-type: none;
+}
+
+h3 {
+  margin: 10px 0 10px 5px;
+}
+  
+.resource-cards {
+  @include span-columns(6);
+  @include omega(2n);
+  background-color: $light-tan;
+  box-shadow: 10px 10px 30px 0px rgba(0,0,0,0.75);
+  height: 250px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+  position: relative;
+
+  p {
+    margin: 2px 0 2px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `2px 0` instead of `2px 0 2px 0`